x/gov/spec/02_state.md

```diff
@@ -4,6 +4,48 @@ order: 2
 
 # State
 
+## Proposals
+
+`Proposal` objects are used to tally votes and generally track the proposal's state.
+They contain an array of arbitrary `sdk.Msg`'s which the governance module will attempt
+to resolve and then execute if the proposal passes. `Proposal`'s are identified by a
+unique id and contains a series of timestamps: `submit_time`, `deposit_end_time`,
+`voting_start_time`, `voting_end_time` which track the lifecycle of a proposal
+
++++ https://github.com/cosmos/cosmos-sdk/blob/v0.40.0/proto/cosmos/gov/v1beta2/gov.proto#L55-L77
+
+A proposal will generally require more than just a set of messages to explain its
+purpose but need some greater justification and allow a means for interested participants
+to discuss and debate the proposal. In most cases, it is encouraged to have an off-chain
+system that supports the on-chain governance process. To accommodate for this, a
+proposal contains a special `metadata` field, an array of bytes, which can be used to
+add context to the proposal. The `metadata` field allows custom use for networks, however,
+it is expected that the field contain a URL or some form of CID using a system such as
+[IPFS](https://docs.ipfs.io/concepts/content-addressing/). To support the case of
+interoperability across networks, the SDK recommends that the `metadata` resolves into
```

Comment:
```suggestion
interoperability across networks, the SDK recommends that the `metadata` represents
```

imo "resolves into" sounds like the metadata bytes field is an URL to a JSON file